layers/+emacs/org/README.org

```diff
@@ -569,6 +570,9 @@ variable =org-enable-asciidoc-support= to =t=.
     '((org :variables org-enable-asciidoc-support t)))
 #+END_SRC
 
+** Mixed pitched support
+Set the variable =org-enable-mixed-pitch= to =t=.
```

Comment:
Please also add a warning/note about the interaction with company.
See `Interaction with company` section on its repository.